run.py

```diff
@@ -282,7 +219,7 @@ def check_requirement(check_func):
   if not result:
     print_out('NOT FOUND', name)
     if help_url_id:
-      print 'Please see %s%s' % (REQUIREMENTS_URL, help_url_id)
+      print('Please see %s%s' % (REQUIREMENTS_URL, help_url_id))
```

Comment:
For changes like this that would work transparently in python27 too, it would be a good idea to just make them in master, if @lipis is amenable...?